app/src/c/main.c

```diff
@@ -115,12 +120,15 @@ static void prv_update_app_glance(AppGlanceReloadSession *session,
       APP_LOG(APP_LOG_LEVEL_ERROR, "AppGlance Error: %d", result);
     }
     t = expiration_time;
-  }
 }
 
 static void refresh_ui() {
+  APP_LOG(APP_LOG_LEVEL_DEBUG, "Refresh ui");
   time_t current_time = time(NULL);
-  strftime(s_app_buffer, sizeof s_app_buffer, s_app_fmt, localtime(&current_time));
+  static char s_fmt[255];
+  strcpy( s_fmt, "" );
+  snprintf( s_fmt, sizeof( s_fmt ), "%s\n%s", s_app_fmt, s_app_time_fmt );
```

Comment:
So, this string will now always be 2lines. I worry that it is centered differently now. Could you check the display with empty second line vs one line?

Reply:
![pebble_screenshot_2018-03-17_18-19-18](https://user-images.githubusercontent.com/21126513/37557049-ca0f0602-2a0f-11e8-9c4b-798d36310ef6.png)
![pebble_screenshot_2018-03-17_18-19-30](https://user-images.githubusercontent.com/21126513/37557050-ca4ff914-2a0f-11e8-9cb6-88bd0ae67842.png)
Looks good. Second line isn't display if empty because have no symbols.